docs/integrations/databricks.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Databricks'
+---
+
+## Partial Support
+
+Currently, Elementary only supports Dbt artifacts on Databricks.
+
```

Comment:
I don't think this explains it to the user. 
(Also, dbt is always lower case.) 

I would say:
"
Currently, Elementary Databricks integration is partial, and supports:
1. Installing the dbt package to upload dbt artifacts, test results and run results.
2. Data observability report generation.
3. Slack alerts on failures in models and tests. 

**Elementary anomaly detection tests are not supported yet**.
"